app/src/queries/markets_home.ts

```diff
@@ -75,11 +78,16 @@ export const queryMyMarkets = gql`
   ${MarketDataFragment}
 `
 
-type buildQueryType = MarketFilters & { whitelistedCreators: boolean; whitelistedTemplateIds: boolean }
+type buildQueryType = MarketFilters & {
+  whitelistedCreators: boolean
+  whitelistedTemplateIds: boolean
+  context: Maybe<ConnectedWeb3Context>
+}
```

Comment:
I think you can use the networkId as prop, or maybe put the timeout as a variable instead of context, I think the first option is better.